sensor/kubernetes/certrefresh/utils.go

```diff
@@ -0,0 +1,89 @@
+package certrefresh
```

Comment:
Could we find a better name for the file than utils?

Reply:
Renamed to `sensor_owner_ref.go`